lib/mavenlink/workspace_allocation.rb

```diff
@@ -1,4 +1,9 @@
 module Mavenlink
   class WorkspaceAllocation < Model
+    def split_allocation(date)
+      response = client.put("#{collection_name}/split", split_date: date, workspace_allocation_id: id)
+      update_attributes(response["workspace_allocations"].values.first)
```

Comment:
Here `update_attributes` will make another PUT call to update the remote record that we can avoid. We should be able to avoid that with just reassigning `self.attributes`.

```suggestion
      self.attributes = response["workspace_allocations"].values.first
```